### ISSUE TITLE
Tutorial / Comments

### ISSUE THREAD

**Issue by cospin** [joshblack/elixir-graphql-boilerplate#1]
Hello! I'm a little new to Relay, GraphQL and Elixir. After much searching I found myself with this boilerplate, it seems a fantastic job.

Have you thought about adding some comments or doing a tutorial on this boilerplate? For example, things like `/connection/connection.ex` I do not have much idea what it is, etc.

The `Team.all/2` and `Team.find/2` method also seems to be missing.

Thanks!

**Comment by joshblack** [comment]
Hi @CristianOspina!

So far this has just been a template for setting up a couple of my own projects that I've been using, so sorry about anything that's missing 😅 

I'm consider doing some videos on how to set up GraphQL+Relay with this Elixir boilerplate for Egghead, but am still trying to gauge interest. (I currently have a course up there that uses the JS version: https://egghead.io/courses/build-a-graphql-server)

If you're interested though, I could definitely put up some resources to help out with this project!

**Comment by cospin** [comment]
Of course! In recent weeks I have been very interested in GraphQL + Relay + Elixir. But it has been made very difficult by the lack of information. Egghead is a bit expensive for my budget and only for one or two courses it's really hard. But if you can do something like the course you have of JS, but in Elixir, it would be more than perfect. Or at least some sort of comparison, I found a lot of information from GraphQL with Node, but many times I do not know how to do it in Elixir. Thank you!

**Comment by joshblack** [comment]
Sounds good! All the points you make are totally fair 😄 

At the least I'll try and just do a simple write-up on usage for this boilerplate and publish it on something like Medium. Hope that works!
